Задание 4.
В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. Какова вероятность того, что все мячи белые? Какова вероятность того, что ровно два мяча белые? Какова вероятность того, что хотя бы один мяч белый?

Все мячи белые - это из каждой корзины вытащили по 2 белых.

2 белых из первой корзины.

In [1]:
P1 = (7 / 10) * (6 / 9)
P1

0.4666666666666666

2 белых из второй корзины.

In [2]:
P2 = (9 / 11) * (8 / 10)
P2

0.6545454545454547

Общая вероятность.

In [4]:
P = P1 * P2
P

0.3054545454545455

Получаем 0.305 или 30.5%, что из каждого ящика достанут по 2 белых шара.

В следующем условии, что ровно 2 мяча белые могут быть несколько комбинаций: 2+0, 1+1, 0+2.

Рассмотрим 2 + 0.

In [27]:
P1 = (7 / 10) * (6 / 9)
P1

0.4666666666666666

In [28]:
P2 = (2 / 11) * (1 / 10)
P2

0.018181818181818184

In [29]:
P_20 = P1 * P2
P_20

0.008484848484848486

Рассмотрим 1 + 1.

In [30]:
P1 = 7 / 10
P1

0.7

In [31]:
P2 = 9 / 11
P2

0.8181818181818182

In [32]:
P_11 = P1 * P2
P_11

0.5727272727272728

Рассмотрим 0 + 2.

In [33]:
P1 = (3 / 10) * (2 / 9)
P1

0.06666666666666667

In [34]:
P2 = (9 / 11) * (8 / 10)
P2

0.6545454545454547

In [35]:
P_02 = P1 * P2
P_02

0.04363636363636364

Теперь складываем вероятности всех трех комбинаций.

In [36]:
P = P_20 + P_11 + P_02
P

0.624848484848485

Получаем 0.625 или 62.5%, что всего достанут 2 белых шара.

Далее, что хотя бы 1 мяч белый, дает нам много различных комбинаций. Решим от обратного.

Найдем вероятность того, что все мячи, которые вытащили - черные.

In [37]:
P1 = (3 / 10) * (2 / 9)
P1

0.06666666666666667

In [38]:
P2 = (2 / 11) * (1 / 10)
P2

0.018181818181818184

In [39]:
P = P1 * P2
P

0.0012121212121212123

In [41]:
1 - P

0.9987878787878788

Получаем 0.999 или 99.9%, что хотя бы 1 шар будет белый.